### PR TITLE
changed export .zip to tar.gz to not lie about it. but kept backwards…

### PR DESCRIPTION
**Pull request by schlenkibus**
… compability in boled and webui

### CODE REVIEW

File: playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/BankEditButtonMenu.cpp [hunk 1]
```diff
@@ -125,7 +125,8 @@ BankEditButtonMenu::FileInfos BankEditButtonMenu::extractFileInfos(std::experime
 bool BankEditButtonMenu::applicableBackupFilesFilter(std::experimental::filesystem::directory_entry term)
 {
   auto fileName = term.path().filename().string();
-  return fileName.find(".xml") == Glib::ustring::npos ? true : fileName.find(".xml.zip") != Glib::ustring::npos;
+  return fileName.find(".xml") == Glib::ustring::npos || fileName.find(".xml.zip") != Glib::ustring::npos &&
+																															 fileName.find(".xml.tar.gz") != Glib::ustring::npos;
```
Comment (hhoegelo) [inline playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/BankEditButtonMenu.cpp]: I doubt this will compile.

**Comment by schlenkibus** [comment]
Reformatting and general Cleanup

File: playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/BankEditButtonMenu.cpp [hunk 2]
```diff
@@ -114,18 +110,13 @@ void BankEditButtonMenu::newBank()
...
-  return infos;
+  return FileInfos{file};
 }
 
 bool BankEditButtonMenu::applicableBackupFilesFilter(std::experimental::filesystem::directory_entry term)
 {
   auto fileName = term.path().filename().string();
-  return fileName.find(".xml") == Glib::ustring::npos ? true : fileName.find(".xml.zip") != Glib::ustring::npos;
+  return fileName.find(".xml") == Glib::ustring::npos || fileName.find(".xml.zip") != Glib::ustring::npos && fileName.find(".xml.tar.gz") != Glib::ustring::npos;
 }
```
Comment (hhoegelo) [inline playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/BankEditButtonMenu.cpp]: still I don't like the style of this line. How about at least some braces and line breaks? But still, as a reader, I do not understand why xml files qualify, whereas xml.zip and xml.tar.gz files do not. Maybe some intermediate vars like 

auto isXmlFile = ...;
auto isFooBar = ...;

auto mightBeBackup = logical combination of determined file types

File: playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/BankEditButtonMenu.cpp [hunk 3]
```diff
@@ -114,18 +110,15 @@ void BankEditButtonMenu::newBank()
...
+  return FileInfos{file};
 }
 
 bool BankEditButtonMenu::applicableBackupFilesFilter(std::experimental::filesystem::directory_entry term)
 {
   auto fileName = term.path().filename().string();
-  return fileName.find(".xml") == Glib::ustring::npos ? true : fileName.find(".xml.zip") != Glib::ustring::npos;
+  auto fileIsBank = fileName.find(".xml") != Glib::ustring::npos;
+  auto fileIsBackup = fileName.find(".xml.zip") != Glib::ustring::npos || fileName.find(".xml.tar.gz") != Glib::ustring::npos;
+  return !(fileIsBank && !fileIsBackup);
```
Comment (hhoegelo) [inline playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/BankEditButtonMenu.cpp]: sorry for nitpicking: I don't understand why we need to test it is 'not a bank' - we do not test it is 'not a word document' and 'not a pizza' - wouldn't it be sufficient to return fileIsBackup? Maybe I am wrong, though.

Comment (schlenkibus) [reply playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/BankEditButtonMenu.cpp]: you are right. fix will be following

**Pull request merged**